src/store/selectors.ts

```diff
@@ -0,0 +1,6 @@
+import {AppRootStateType} from '../types/types';
+import {useSelector} from 'react-redux';
+
+export const isLoggedIn = useSelector<AppRootStateType, boolean>(
```

Comment:
Selectors should be named `______Selector`, with a word 'Selector' at the end for clarity. This way when reading code you'll clearly understand, that a certain component uses data from Redux store and it is not a random function.

Reply:
rename `isLoggedIn` to `isLoggedInSelector`